documentation/release_notes.markdown

```diff
@@ -4,87 +4,271 @@ title: "Puppet Server: Release Notes"
 canonical: "/puppetserver/latest/release_notes.html"
 ---
 
+## Puppet Server 1.1.0
```

Comment:
We'd talked about this a bit with @nfagerlund and can't remember exactly where we landed.  I think we want to leave only the 1.x release notes on the stable branch and only the  2.x release notes on the master branch.  If true, we probably want to just leave this file alone as part of the merge-up.  At the very least, we should preserve the Puppet Server 2.1 notes whereas it looks this PR as it stands would wipe them out.  @nfagerlund - WDYT?